pkg/crc/machine/bundle/repository.go

```diff
@@ -84,7 +84,7 @@ func (repo *Repository) Use(bundleName string) (*CrcBundleInfo, error) {
 	if err != nil {
 		return nil, err
 	}
-	if bundleInfo.IsOpenShift() {
+	if !bundleInfo.IsPodman() {
```

Comment:
I though of making change in the `createSymlinkOrCopyOpenShiftClient` and only have this operation in case the `oc` binary present in the bundle otherwise don't do anything but then it means if we miss the `oc` binary in the bundle then there is no error out.

Reply:
There will be an error in `verify()`, I don't think we should be hardcoding "bundle X must have oc" in too many places to avoid this bug happening again in the future.